### ISSUE TITLE
Long loading time in 2.0.5

### ISSUE THREAD

**Issue by trungleduc** [jupyterlab/jupyter-collaboration#263]
## Description

We are using `jupyter-collaboration` in `jupytercad`, in 2.0.5, reloading the page with an opened document causes a very long loading time. Below is the comparison between 2.0.4 and 2.0.5

- 2.0.4
 
![2 0 4](https://github.com/jupyterlab/jupyter-collaboration/assets/4451292/f4363f07-8e81-4b4b-ad5e-ab84a7e888a7)

- 2.0.5

https://github.com/jupyterlab/jupyter-collaboration/assets/4451292/ee9aecd3-b1ff-4c2f-a63d-0d5ce0318cc0

The same issue happens with the notebook files.

## Reproduce

Open a notebook then refresh JupyterLab

## Context

```
jupyter-collaboration     2.0.5              pyhd8ed1ab_0    conda-forge
jupyter-lsp               2.2.4              pyhd8ed1ab_0    conda-forge
jupyter_client            8.6.1              pyhd8ed1ab_0    conda-forge
jupyter_core              5.7.2           py312h7900ff3_0    conda-forge
jupyter_events            0.10.0             pyhd8ed1ab_0    conda-forge
jupyter_server            2.13.0             pyhd8ed1ab_0    conda-forge
jupyter_server_fileid     0.9.1              pyhd8ed1ab_0    conda-forge
jupyter_server_terminals  0.5.3              pyhd8ed1ab_0    conda-forge
jupyter_ydoc              2.0.1              pyhd8ed1ab_0    conda-forge
jupytercad-app            2.0.0a2                  pypi_0    pypi
jupytercad-core           2.0.0a2                  pypi_0    pypi
jupytercad-lab            2.0.0a2                  pypi_0    pypi
jupyterlab                4.1.5              pyhd8ed1ab_0    conda-forge
jupyterlab_pygments       0.3.0              pyhd8ed1ab_1    conda-forge
jupyterlab_server         2.25.4             pyhd8ed1ab_0    conda-forge
```

<details><summary>Troubleshoot Output</summary>
<pre>
Paste the output from running `jupyter troubleshoot` from the command line here.
You may want to sanitize the paths in the output.
</pre>
</details>

<details><summary>Command Line Output</summary>
<pre>
Paste the output from your command line running `jupyter lab` here, use `--debug` if possible.
</pre>
</details>

<details><summary>Browser Output</summary>

<pre>
Paste the output from your browser Javascript console here, if applicable.

</pre>
</details>

**Comment by krassowski** [comment]
If we cannot release just yet (https://github.com/jupyterlab/jupyter-collaboration/issues/265), should we yank 2.0.5?

**Comment by davidbrochart** [comment]
Hard to say, the bug addressed by #255 is more severe but less frequent, and the long loading time probably more frequent but just annoying.
